docs/docs/recipes/organizations/configure-organizations.mdx

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 2
+---
+
+# Configure organizations
+
+## Configure via Console
+
+We'll go through the process of configuring the organizations feature via Logto Console (https://cloud.logto.io/).
+
+### Start setting up your organization feature
+
+When you activate the organization feature, you'll be guided through essential settings like setting up organization permissions, roles, and creating your first organization. You will gain a clearer understanding of how organizations function within Logto. After the initial setup, you can add members, assign roles within the organization, or refine your organization template.
+
+In the organization list, you can create an organization and configure its details. On the organization details page, you can:
+
+1. Modify the organization's name and description.
+2. Add members and assign them organization roles.
+3. Remove a user's membership in the organization.
+4. Delete the organization.
+5. Access a guide to understand more about organizations and the organization template.
+
+### Adding members and assigning organization roles
+
+Users can hold one or more roles. When adding members to an organization, you have the option to assign roles to multiple users at once. If you leave this assignment blank, the added users will not receive any roles.
+
+In the user management section, on the user detail page, you can see which organizations the users belong to and what organization roles they have.
+
+### Configure organization template
+
+The organization template consists of permissions and roles, which can be edited freely; any changes will affect users with those roles across all organizations.
```

Comment:
Should we swap these two paragraphs? Logically we can only create permissions and roles in template before being able to adding members and assigning organization roles, right?

Reply:
right